### DISCUSSION TITLE
How to add a separate custom text for each sources

### DISCUSSION THREAD

**Discussion by sankajalakshan** [aahnik/tgcf#169] [General]
How can we add a separate custom text for each sources without enabling the "show_forwarded_from".
Eg- 
Original message from source A = test1
Customized forwarding message = test1 form source A

Original message from source B = test2
Customized forwarding message = test2 form source B

**Comment by tissole** [comment]
If I understand correctly you want all your messages to be captioned with the source, but without the drawback of the "forwarded from" tag. I don't think that now is possible. There are 3 other requests regarding captioning, [40](https://github.com/aahnik/tgcf/discussions/40#discussion-3289497), [55](https://github.com/aahnik/tgcf/issues/55#issue-862719793) and [86](https://github.com/aahnik/tgcf/discussions/86#discussion-3335648), related to captioning files with their names and other attributes, or removing the captions altogether. You want a third option captioning with the source. 

There are many features waiting to be implemented, maybe this will be too, it has a certain use if a user wants to remember where the files originated from.

**Comment by aahnik** [comment]
This is moved to #177
